filters/HAGFilter.cpp

```diff
@@ -56,6 +59,17 @@ std::string HAGFilter::getName() const
     return s_info.name;
 }
 
+void HAGFilter::addArgs(ProgramArgs& args)
+{
+    args.add("count", "The number of points to fetch to determine the ground point [default: 1].",
+            m_count, point_count_t(1));
+    args.add("max_distance", "The maximum distance to the farthest nearest neighbor before the height above ground is not calculated [default: 0 (disabled)]", m_max_distance);
+    args.add("allow_extrapolation", "If true and count > 1, allow extrapolation [default: true].",
+            m_allow_extrapolation, true);
+    args.add("delaunay_fans", "Construct local Delaunay fans and infer heights from them [default: false].",
```

Comment:
Very minor detail, but I wonder if just `delaunay` may be easier to remember (and less to type) than `delaunay_fans`.

Reply:
Okay, will change

Reply:
Okay, addressed